easypost/requestor.py

```diff
@@ -48,24 +58,18 @@ def request_raw(self, method, url, params=None, api_key_required=True):
             )
 
         abs_url = "%s%s" % (api_base, url or "")
-        params = self._objects_to_ids(params)
+        params = self._objects_to_ids(param=params or {})
 
         ua = {
             "client_version": VERSION,
             "lang": "python",
             "publisher": "easypost",
             "request_lib": request_lib,
+            "lang_version": platform.python_version(),
+            "platform": platform.platform(),
+            "uname": " ".join(platform.uname()),
+            "implementation": platform.python_implementation(),
         }
-        for attr, func in (
```

Comment:
Why was this refactored here? By doing so we lose the try/except that may have been needed in some situations

Reply:
Reverted in 94d6bd1b26f8d68dcb940fd8f3f37d758f270568

Reply:
> > @nwithan8 lint needs to be fixed.
> 
> It's related to this: [#163 (comment)](https://github.com/EasyPost/easypost-python/pull/163#discussion_r830366136) `mypy` doesn't like the fact that we're just executing a function, with no concept of what said function will return. We'll need to decide how to handle this.

Let's get the build passing in the most compatible way to what was there unless we can prove the cold is old and unnecessary anymore.

Reply:
Resolved in [f20ed30](https://github.com/EasyPost/easypost-python/pull/163/commits/f20ed308b210f3a207f8c94fa1eac9b6ef2ceefe)